discovery-infra/test_infra/utils.py

```diff
@@ -953,4 +953,10 @@ def resource_param(value, param_name, operators):
     try:
         return sum((consts.OPERATOR_PARAMS[operator][param_name] for operator in operators), value)
     except KeyError as e:
-        raise ValueError(f"Unknown operator name {e.args[0]}")
\ No newline at end of file
+        raise ValueError(f"Unknown operator name {e.args[0]}")
+
+def ocs_resource_param(value, param_name, operators):
+    if "ocs" in operators and value > consts.OPERATOR_PARAMS["ocs"][param_name]:
```

Comment:
How would it work with multiple OLMs? Do we need sum of all the requirements or just getting max?
We can use `resource_param` to receive a function

```
def resource_param(value, param_name, operators, func):
    try:
        return func((consts.OPERATOR_PARAMS[operator][param_name] for operator in operators), value)
    except KeyError as e:
        raise ValueError(f"Unknown operator name {e.args[0]}")
```

Reply:
We need to sum up all the requirements

Reply:
So I think `resource_param` already takes care of that logic. no?

Reply:
Assuming we add worker_count and worker_disk to other operator we can reuse.